indico/queries/submission.py

```diff
@@ -33,57 +33,95 @@ class ListSubmissions(PagedRequest):
+                workflowIds: $workflowIds
+                filters: $filters
                 limit: $limit
-                orderBy: $orderBy,
-                desc: $desc,
+                orderBy: $orderBy
+                desc: $desc
                 after: $after
-
-            ){
+            ) {
                 submissions {
+                id
+                datasetId
+                workflowId
+                status
+                createdAt
+                updatedAt
+                createdBy
+                updatedBy
+                completedAt
+                errors
+                filesDeleted
+                inputFiles {
                     id
-                    datasetId
-                    workflowId
-                    status
-                    inputFiles {
-                        id
-                        filename
-                        filepath
-                        filetype
-                        fileSize
-                        numPages
-                    }
-                    inputFile
-                    inputFilename
-                    resultFile
-                    deleted
-                    retrieved
-                    errors
-                    reviews {
-                        id
-                        createdAt
-                        createdBy
-                        completedAt
-                        rejected
-                        reviewType
-                        notes
-                    }
+                    filepath
+                    filename
+                    filetype
+                    submissionId
+                    fileSize
+                    numPages
+                }
+                inputFile
+                inputFilename
+                resultFile
+                outputFiles {
+                    id
+                    filepath
+                    submissionId
+                    componentId
+                    createdAt
+                }
+                retrieved
+                autoReview {
+                    id
+                    submissionId
+                    createdAt
+                    createdBy
+                    startedAt
+                    completedAt
+                    rejected
+                    reviewType
+                    notes
+                    changes
+                }
+                retries {
+                    id
+                    submissionId
+                    previousErrors
+                    previousStatus
+                    retryErrors
+                }
+                AutoReviewLoaded
+                ocrEngine
+                reviews {
+                    id
+                    submissionId
+                    createdAt
+                    createdBy
+                    startedAt
+                    completedAt
+                    rejected
+                    reviewType
+                    notes
+                    changes
```

Comment:
I'm conflicted about including `changes` - the payload could be massive so maybe we leave this off for now and now/future add a different `GetReviews` query for submissions that include changes

Reply:
Sounds good I'm opting for just removing changes since class `SubmissionReviews` does not include it by default, and will create a ticket for a `GetReviews` query to be able to access changes given a review id

Reply:
Ticket here: https://indicodata.atlassian.net/browse/DEV-11438